### ISSUE TITLE
Missing properties from type 'Location': startToken, endToken, source

### ISSUE THREAD

**Issue by jimmybaker** [dotansimha/graphql-code-generator#1599]
**Describe the bug**
I'm using the typescript-react-apollo plugin to generate components from my queries. The Generated component looks like this:

<img width="424" alt="Image 2019-03-28 at 1 54 22 PM" src="https://user-images.githubusercontent.com/50668/55184788-0b040080-5161-11e9-9e1e-eddf92535797.png">

But, notice how the `loc` key in that object has a red underline. It's failing because Typescript says that it's missing the following properties:

<img width="870" alt="Image 2019-03-28 at 1 37 42 PM" src="https://user-images.githubusercontent.com/50668/55184843-2707a200-5161-11e9-8939-37ac9dcfc90e.png">

**To Reproduce**

1. My GraphQL schema:

```graphql
# My schema is way to big to put in here.
```

2. My GraphQL operations:

```graphql
# Put your operations here
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: "schema.graphql"
documents: ["src/graphql/queries/**/*.graphql"]
generates:
  src/definitions/graphql.tsx:
    plugins:
      - "typescript"
      - "typescript-operations"
      - "typescript-react-apollo"
      - "fragment-matcher"
    config:
      withHOC: false
      withComponent: true
      noGraphQLTag: true
      typesPrefix: I
  ./graphql.schema.json:
    plugins:
      - "introspection"

```

**Expected behavior**
I expected to not have a TypescriptError

**Environment:**

- OS: macOS 10.14.3
- "@graphql-codegen/cli": "^1.0.6",
- "@graphql-codegen/fragment-matcher": "1.0.6",
- "@graphql-codegen/introspection": "^1.0.6",
- "@graphql-codegen/typescript": "1.0.6",
- "@graphql-codegen/typescript-operations": "1.0.6",
- "@graphql-codegen/typescript-react-apollo": "1.0.6", 
- NodeJS: 10.14.0

**Additional context**
There is a Location object in my graphql schema, but I've set the type prefix to `I` so I don't think it's a naming collision. (Any startToken, endToken, and source aren't properties of my schema's Location)

**Comment by dotansimha** [comment]
Hi @jimmybaker ,
I don't think it's a naming collision, it seems like a `graphql` typings issue.  Which version of `graphql` and `@types/graphql` do you use? 
Because it seems like the most updated version of `@types/graphql` supports `loc` property: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/graphql/language/ast.d.ts#L186-L190 (it's optional, so maybe we don't have to include it in the generated output, not sure).

**Comment by CameronFletcher92** [comment]
I'm seeing the same things using:
 "@types/graphql": "14.2.0"
and
"graphql": "14.2.1",

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/1663

**Comment by dotansimha** [comment]
Fixed in 1.0.7 :)

**Comment by jimmybaker** [comment]
Excellent. Thank you!
